src/relay/backend/te_compiler_cache.cc

```diff
@@ -754,6 +754,24 @@ CachedFunc ShapeFuncFor(const Function& prim_func, const Target& target,
   return MakeShapeFunc().Create(prim_func, target, renamer);
 }
 
+std::pair<Array<te::Tensor>, std::string> LowerTECompute(const Function& source_func, Target target,
+                                                         bool return_inputs) {
+  LowerToTECompute lower_te_compute(target);
+  auto outputs = lower_te_compute.Lower(source_func, [&](std::string name) { return name; });
```

Comment:
nit: just nitpick again :-)

```suggestion
  Array<te::Tensor> outputs = lower_te_compute.Lower(source_func, [&](std::string name) { return name; });
```